liquibase-standard/src/main/java/liquibase/sqlgenerator/core/CreateSequenceGenerator.java

```diff
@@ -97,9 +97,9 @@ public Sql[] generateSql(CreateSequenceStatement statement, Database database, S
         }
 
         if (statement.getCacheSize() != null) {
-            if (database instanceof OracleDatabase || database instanceof Db2zDatabase || database instanceof PostgresDatabase || database instanceof MariaDBDatabase) {
+            if (database instanceof OracleDatabase || database instanceof Db2zDatabase || database instanceof PostgresDatabase || database instanceof MariaDBDatabase || database instanceof SybaseASADatabase) {
                 if (BigInteger.ZERO.equals(statement.getCacheSize())) {
-                    if (database instanceof OracleDatabase) {
+                    if (database instanceof OracleDatabase || database instanceof SybaseASADatabase) {
```

Comment:
hello @mkarg! 
Could you please verify if the correct syntax for SQL Anywhere is "NO CACHE" instead of "NOCACHE"? I would greatly appreciate your assistance. You can refer to the following link for more information:
https://infocenter.sybase.com/help/index.jsp?topic=/com.sybase.help.sqlanywhere.12.0.1/dbreference/create-sequence-statement.html
cc @filipelautert

Reply:
@rberezen Good catch! According to the manuals, it is `NO CACHE`. I will modify the PR.

Reply:
@rberezen Syntax is corrected in https://github.com/liquibase/liquibase/pull/4417/commits/32a881968982296926472c24d667371bdbb25939, according to latest SAP documentation (https://help.sap.com/docs/SAP_SQL_Anywhere/93079d4ba8e44920ae63ffb4def91f5b/3be47d496c5f1014bf06bb7791ce7a38.html?locale=en-US&q=CREATE%20SEQUENCE). Unit test will follow ASAP.

Reply:
@rberezen Thank you for your kind patience. You can now find the unit test in 6fe525b974821068da73740c8d9ec43a59287627.